### PR TITLE
DEV2-3145: Add bitnami and bitnami/postgres

### PR DESCRIPTION
**Pull request by SimSimY**

### CODE REVIEW

File: charts/fluentd/values.yaml [hunk 1]
```diff
@@ -13,7 +13,7 @@ global:
     organizationSecret: ""
     organizationName: ""
     telemetry:
-      enabled: true
+      enabled: false
```
Comment (assapir) [inline charts/fluentd/values.yaml:16]: is that a related change? shouldn't it be taken from the general value?

Comment (SimSimY) [reply charts/fluentd/values.yaml:16]: It is overrided by the false in `tabnine-cloud` so this is just for constancy, the flag is not used by the helm chart.

**Review by yanir-codota — APPROVED** [review]
